spec/features/admin/order_shipments_links_to_returns_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+RSpec.feature 'Return Button Under Order#edit', type: :feature do
```

Comment:
Only inasmuch as it adds little value at the cost of additional noise. Do you prefer not to have RSpec infer metadata?